### PR TITLE
feat: Add Web3 Failover

### LINKED ISSUE #1047: [FEATURE] Support High-availability General Nodes
**Is your feature request related to a problem? Please describe.**
* When using ALB, there are various problems when nodes are switched in consecutive processing

**Describe the solution you'd like**
* Check the node status at the time of connection and switch to the node, switch alive node.
* refs BoostryJP/ibet-Prime/issues/155

### PR DESCRIPTION
**Pull request by urushibata-k**
close #1047

### CODE REVIEW

File: app/model/blockchain/__init__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+"""
+from .token import (
```
Comment (urushibata-k) [inline app/model/blockchain/__init__.py:19]: With in previous directory configurations, circular references occur.
> contract.py -> web3_utils.py -> node.py(app/model/__init__.py) -> token.py -> contract.py -> ...

File: migrations/versions/022_alter_node.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    meta.bind = migrate_engine
+    try:
+        node = Table("node", meta, autoload=True)
+        Column("endpoint_uri", String(267)).create(node)
+        Column("priority", Integer).create(node)
+
+        # Delete current record
+        con = migrate_engine.connect()
+        query = node.delete().where(node.c.endpoint_uri == None)
+        con.execute(query)
```
Comment (urushibata-k) [inline migrations/versions/022_alter_node.py:37]: Delete existing data(endpoint_uri=null)

File: batch/processor_Block_Sync_Status.py [hunk 3]
```diff
@@ -65,118 +123,138 @@ def append(self, data):
...
-    def peekOldest(self):
+    def peek_oldest(self):
         return self._buffer[self._next]
 
 
-# Watcher
-class Watcher:
-    def __init__(self):
-        pass
+class Processor:
```
Comment (urushibata-k) [inline batch/processor_Block_Sync_Status.py:130]: Almost the same code.
https://github.com/BoostryJP/ibet-Prime/blob/dev-21.12/batch/processor_monitor_block_sync.py

File: app/api/v2/eth.py [hunk 4]
```diff
@@ -207,7 +198,7 @@ def on_post(self, req, res):
                     LOG.error(f"get sender address from signed transaction failed: {err}")
                     continue
                 nonce = int("0x0" if raw_tx[0].hex() == "" else raw_tx[0].hex(), 16)
-                txpool_inspect = GethTxPool.inspect
+                txpool_inspect = web3.geth.txpool.inspect()
```
Comment (urushibata-k) [inline app/api/v2/eth.py:201]: sample execute
```
[-bash ibet-Wallet-API]$ python
Python 3.8.6 (default, Apr 13 2021, 15:15:20) 
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 
>>> from app.utils.web3_utils import Web3Wrapper
>>> 
>>> web3 = Web3Wrapper()
>>> 
>>> result = web3.geth.txpool.inspect()
>>> 
>>> print(result)
AttributeDict({'pending': AttributeDict({}), 'queued': AttributeDict({})})
>>> 
>>> 
```
quorum connected.(not connect ganache-cli)

**Review by YoshihitoAso — APPROVED** [review]

**Pull request merged**